src/__tests__/fireEvent.test.js

```diff
@@ -173,9 +173,7 @@ test('should not fire on disabled TouchableOpacity', () => {
     </TouchableOpacity>
   );
 
-  expect(() => fireEvent.press(screen.getByText('Trigger'))).toThrow(
-    'No handler function found for event: "press"'
-  );
+  expect(() => fireEvent.press(screen.getByText('Trigger'))).not.toThrow();
```

Comment:
we could get rid of the expect altogether here and there
```suggestion
fireEvent.press(screen.getByText('Trigger'));
```